x-pack/plugins/security/public/management/api_keys/api_keys_grid/create_api_key_flyout.tsx

```diff
@@ -83,6 +86,18 @@ const defaultDefaultValues: ApiKeyFormValues = {
     null,
     2
   ),
+  metadata: JSON.stringify(
```

Comment:
I'm torn on this sample metadata. On one hand, it provides a great example of the types of input we accept, but I worry that an inexperienced user would think that this example contains important information, or that these are the only fields available to them.

What do you think about defaulting to an empty JSON object (`{ }`) instead?

Reply:
I agree, the text in the mockup was there just for reference. The link brings users to docs where that example is available, and it is enough to help.

Reply:
Yeh, good point, it's probably a bit too prescriptive - Let's leave it empty.